### ISSUE TITLE
--interpolate-env-vars doesn't work for sqlite parameter, neither is does relative path interpretation

### ISSUE THREAD

**Issue by ugermann** [marian-nmt/marian-dev#616]
### Bug description

The sqlite parameter in a config file isn't interpreted correctly if --relative-paths and / or --interpolate-env-vars is true

### How to reproduce

```
echo ding > train.a
echo dong > train.b
mkdir somepath
echo 'sqlite: ${FOO}.sql' > somepath/config.yml
export FOO=SQLITE
marian -c somepath/config.yml --relative-paths true --interpolate-env-vars true -t train.a train.b -e 1
```

Marian should create somepath/SQLITE.sql. Instead, it creates ${FOO}.sql in the pwd.
The problem is that sqlite isn't part of PATHS in the config parser.

### Context
```
v1.9.1 95c65bb 2020-03-17 03:30:49 +0000
```

* CMake command: 

Irrelevant

* Log file: Attach your training/decoding logs

Add any other information about the problem here.
[train.log](https://github.com/marian-nmt/marian-dev/files/4412158/train.log)

**Comment by snukky** [comment]
PR #612 should fix that, but I will double check the `--interpolate-env-vars` option.

**Comment by snukky** [comment]
Should be fixed now after merging PR #612.
